src/features/controls.ts

```diff
@@ -361,10 +363,14 @@ export const ControlsMixin = (ModelViewerElement:
           this[$promptElement].classList.remove('visible');
         }
 
-        [$onChange]() {
+        [$onChange]({source}: any) {
           this[$deferInteractionPrompt]();
           this[$updateAria]();
           this[$needsRender]();
+
+          if (source === 'user-interaction') {
+            this[$onUserModelOrbit]();
```

Comment:
Let's dispatch an event here. It seems useful, and we can always take it out if we decide it's the wrong API down the road.

Reply:
Yeah I feel like it would be good to sit down and think about all the events that `<model-viewer>` should dispatch. Then in another PR(s) these events could be implemented.

Reply:
It would take a long time to figure out the entire event space in advance. There are a lot of features to go through :)

Using an event here will decouple the related implementations, and it will offer a useful feature to users. We have an excellent event system in the browser, so let's take advantage of it! And, we can definitely discuss walking it back in a future change if it seems like a bad idea.

Reply:
So what you're saying is be bold and ship 😄 I like it.

Should this event just be called `user-orbit-model`? or `user-orbit`?

Should there be an event for `orbit-model` or simply `orbit` when the camera changes not from user input (keyboard/mouse/touch/etc)?

This is why I was suggesting just doing a simple PR after where we could think about what the event should be to make the whole application cohesive.

Reply:
I like us to be bold pre-1.0.0 release. Right now, it's the best time to experiment. It's harder to be bold after you release 1.0.0 :)

It's totally fine to put the change in a subsequent PR if you want to file an issue for further discussion. But, let's limit the scope of the discussion to just this particular event for now.

Reply:
I filed https://github.com/GoogleWebComponents/model-viewer/issues/506 to cover discussion of the event.

Reply:
Can you leave a comment here that we should swap out the Symbol approach with an event via #506 ?